app/classifier/tasks/drawing/markings-renderer.cjsx

```diff
@@ -35,15 +35,14 @@ module.exports = React.createClass
 
   render: ->
     skippedMarks = 0
-
     <g>
-      {for annotation in @props.classification?.annotations ? []
+      {for annotation in @props.classification?.annotations ? []      
         annotation._key ?= Math.random()
         isPriorAnnotation = annotation isnt @props.annotation
         taskDescription = @props.workflow.tasks[annotation.task]
         if taskDescription.type is 'drawing'
           <g key={annotation._key} className="marks-for-annotation" data-disabled={isPriorAnnotation || null}>
-            {for mark, i in annotation.value when parseInt(mark.frame) is parseInt(@props.frame)
+            {for mark, i in annotation.value when @props.workflow.configuration.multi_image_clone_markers or parseInt(mark.frame) is parseInt(@props.frame)
```

Comment:
Makes sense, thanks. It's worth noting that, by the same reasoning, we should probably change these lines as well: https://github.com/zooniverse/Panoptes-Front-End/blob/3f0142adc2a5cb4501da8164463ad888a56ae068/app/components/multi-image-subject-options-editor.cjsx#L27 and https://github.com/zooniverse/Panoptes-Front-End/blob/3f0142adc2a5cb4501da8164463ad888a56ae068/app/components/multi-image-subject-options-editor.cjsx#L41. 
The `?` should be after `workflow`, not `configuration`.